hadoop-hdfs-project/hadoop-hdfs-client/src/main/java/org/apache/hadoop/hdfs/DeadNodeDetector.java

```diff
@@ -396,13 +395,13 @@ private void probeCallBack(Probe probe, boolean success) {
             probe.getDatanodeInfo());
         removeDeadNode(probe.getDatanodeInfo());
       } else if (probe.getType() == ProbeType.CHECK_SUSPECT) {
-        LOG.debug("Remove the node out from suspect node list: {}.",
+        LOG.info("Remove the node out from suspect node list: {}.",
```

Comment:
when a lot of invalid relicas,  it should have many supsect nodes  but not dead nodes.
These nodes all will print this log.
What is the purpose of printing this log? 
The client can access normally the suspect node.

Reply:
ditto